### PR TITLE
chore: Replace str.format() and string concatenation in samples with f-string

### LINKED ISSUE #164: Replace str.format() and string concatenation in samples with f-string
In PR https://github.com/googleapis/python-retail/pull/160, there is a review comment requesting to replace all str.format() or string concatenation in samples with an f-string. I'm going to track this as a separate work item because it affects all retail samples. See https://github.com/googleapis/python-retail/pull/160#discussion_r812097695

### PR DESCRIPTION
**Pull request by t-karasova**
Fixes #164 🦕

### CODE REVIEW

**Review by kweinmeister — APPROVED** [review]

**Review by parthea — APPROVED** [review]
LGTM

File: samples/interactive-tutorials/product/set_inventory.py [hunk 1]
```diff
@@ -50,7 +50,7 @@ def get_product_with_inventory_info(product_name: str) -> Product:
 
     fulfillment_info = FulfillmentInfo()
     fulfillment_info.type_ = "pickup-in-store"
-    fulfillment_info.place_ids = ["store1", "store2"]
+    fulfillment_info.place_ids = ["store1", "store3"]
```
Comment (parthea) [inline samples/interactive-tutorials/product/set_inventory.py:53]: revert unrelated change
```suggestion
    fulfillment_info.place_ids = ["store1", "store2"]
```

File: samples/interactive-tutorials/events/rejoin_user_event.py [hunk 2]
```diff
@@ -24,16 +24,15 @@
...
+default_catalog = f"projects/{project_id}/locations/global/catalogs/default_catalog"
+
 visitor_id = "test_visitor_id"
 
 
 # get rejoin user event request
 def get_rejoin_user_event_request():
     # TO CHECK THE ERROR HANDLING TRY TO PASS INVALID CATALOG:
-    # default_catalog = "projects/{0}/locations/global/catalogs/invalid_catalog".format(project_number)
+    # default_catalog = f"projects/{project_number}/locations/global/catalogs/invalid_catalog"
```
Comment (chenlei1216) [inline samples/interactive-tutorials/events/rejoin_user_event.py:35]: project_id

File: samples/interactive-tutorials/test_resources_recovery/create_test_resources.py [hunk 3]
```diff
@@ -50,16 +50,14 @@ def create_bucket(bucket_name: str) -> Bucket:
...
         return storage_client.bucket(bucket_name)
     else:
         bucket = storage_client.bucket(bucket_name)
         bucket.storage_class = "STANDARD"
         new_bucket = storage_client.create_bucket(bucket, location="us")
         print(
-            "Created bucket {} in {} with storage class {}".format(
-                new_bucket.name, new_bucket.location, new_bucket.storage_class
-            )
+            "Created bucket {new_bucket.name} in {new_bucket.location} with storage class {new_bucket.storage_class}"
```
Comment (chenlei1216) [inline samples/interactive-tutorials/test_resources_recovery/create_test_resources.py:60]: missing f?

**Review by chenlei1216 — APPROVED** [review]

File: samples/interactive-tutorials/events/rejoin_user_event.py (hunk 2 above)
Comment (parthea) [reply samples/interactive-tutorials/events/rejoin_user_event.py:35]:
```suggestion
    # default_catalog = f"projects/{project_id}/locations/global/catalogs/invalid_catalog"
```

File: samples/interactive-tutorials/test_resources_recovery/create_test_resources.py (hunk 3 above)
Comment (parthea) [reply samples/interactive-tutorials/test_resources_recovery/create_test_resources.py:60]:
```suggestion
            f"Created bucket {new_bucket.name} in {new_bucket.location} with storage class {new_bucket.storage_class}"
```

File: samples/interactive-tutorials/events/write_user_event.py [hunk 4]
```diff
@@ -51,8 +50,7 @@ def get_user_event():
 # get write user event request
 def get_write_event_request(user_event):
     # TO CHECK THE ERROR HANDLING TRY TO PASS INVALID CATALOG:
-    # default_catalog = "projects/{0}/locations/global/catalogs/invalid_catalog"
-    # .format(project_number)
+    # default_catalog = f"projects/{project_number}/locations/global/catalogs/invalid_catalog"
```
Comment (parthea) [inline samples/interactive-tutorials/events/write_user_event.py:53]:
```suggestion
    # default_catalog = f"projects/{project_id}/locations/global/catalogs/invalid_catalog"
```
